### PR TITLE
Some smaller Arm cleanups/improvements

### PR DESCRIPTION
**Pull request by webmaster128**
Follow up of https://github.com/CosmWasm/rust-optimizer/pull/56

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ executors:
 jobs:
   build-amd64:
     executor: ubuntu-amd64
+    environment:
+      BUILDKIT_PROGRESS: plain
```
Comment (maurolacy) [inline .circleci/config.yml:16]: Good find. I was wondering how to do that.

File: Makefile [hunk 2]
```diff
@@ -28,7 +28,7 @@ build-workspace-optimizer-x86_64: use-rust-optimizer-multi
 	docker buildx build --platform linux/amd64 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
 
 build-workspace-optimizer-arm64: use-rust-optimizer-multi
-	docker buildx build --platform linux/arm64 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
+	docker buildx build --platform linux/arm64/v8 -t $(DOCKER_NAME_WORKSPACE_OPTIMIZER):$(DOCKER_TAG) --target workspace-optimizer --load .
```
Comment (maurolacy) [inline Makefile:31]: 👍🏼  I hope this was compatible / equivalent.

**Review by maurolacy — APPROVED** [review]
Great.

File: Makefile (hunk 2 above)
Comment (webmaster128) [reply Makefile:31]: Alright, I wait for your confirmation then.

I changed this for consistency.

Comment (maurolacy) [reply Makefile:31]: It seems `linux/arm64/v8` is (currently) just a synonym of `linux/arm64`.

Comment (webmaster128) [reply Makefile:31]: okay, great. Then we have the same everywhere now.

**Pull request merged**